_content/monographs/what-buddhism-is_khin-u-ba.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "What Buddhism Is"
+authors:
+  - "U Ba Khin"
+external_url: "https://ia803205.us.archive.org/13/items/what-buddhism-is_202008/What-Buddhism-Is.pdf"
+drive_links:
```

Comment:
Go ahead and add the Pariyatti link as source_url and add a `# note` that their download is broken as of March 2025